diff --git a/deps/igraph-0.10.x/CMakeLists.txt b/deps/igraph-0.10.x/CMakeLists.txt
index 727398e..5b49141 100644
--- a/deps/igraph-0.10.x/CMakeLists.txt
+++ b/deps/igraph-0.10.x/CMakeLists.txt
@@ -19,7 +19,7 @@ ExternalProject_Add(igraph_0_10
   PREFIX "${IGRAPH_INSTALL}"
   URL "https://github.com/igraph/igraph/releases/download/0.10.12/igraph-0.10.12.tar.gz"
   SOURCE_DIR "${IGRAPH_DOWNLOAD}"
-  CMAKE_ARGS "-DCMAKE_INSTALL_PREFIX=${IGRAPH_INSTALL}" "-DBUILD_SHARED_LIBS=ON" "-DCMAKE_INSTALL_LIBDIR=${IGRAPH_INSTALL}/lib" "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}" "-Wno-deprecated" "-DIGRAPH_VERSION=0.10.12"
+  CMAKE_ARGS "-DCMAKE_INSTALL_PREFIX=${IGRAPH_INSTALL}" "-DBUILD_SHARED_LIBS=ON" "-DCMAKE_INSTALL_LIBDIR=${IGRAPH_INSTALL}/lib" "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}" "-Wno-deprecated" "-DIGRAPH_VERSION=0.10.12" "-DCMAKE_COMPILE_WARNING_AS_ERROR=OFF" "-DCMAKE_C_FLAGS=-w -Wno-error" "-DCMAKE_CXX_FLAGS=-w -Wno-error"
   BINARY_DIR "${IGRAPH_BUILD}"
   PATCH_COMMAND echo "No patch for igraph-0.10.x required"
   INSTALL_DIR "${IGRAPH_INSTALL}"
